### ISSUE TITLE
Focus loss when publishing / updating

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#3689]
See #3682

When using a keyboard, publishing or updating a post causes a focus loss. To reproduce:
- create a new post
- use your keyboard
- add some title, add some text in a paragraph
- tab backwards to reach the Publish button 
- press Enter to open the Publish popover
- tab to reach the second Publish button and press Enter to publish

At this point, the popover disappears and since it contained the element that was focused, focus is lost. Tab again and you will see you're forced to start keyboard navigation from the document root.

As mentioned several times, focus losses like this one are one of the worst experiences for keyboard users and should be avoided at all costs.
In this case, focus should be managed programmatically and be moved to a place that makes sense in the editing workflow.

**Comment by mtias** [comment]
Also related to proposed changes in https://github.com/WordPress/gutenberg/issues/3496

**Comment by noisysocks** [comment]
It looks like this was fixed in 1.8.1. Now, after I follow these steps, my focus is on the Update button.
